index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<main id="main">
+  <head> 
+    <meta charset="UTF-8">
+    <title >FCC: Tribute page</title>
+   
+    <link rel="stylesheet" href="style.css">
+  </head>
```

Comment:
- [ ] Your head tag should not be inside the main tag. The head tag contains metadata and should not be joined with the content that will appear on the webpage which is what is inside the body tag. So please move your opening main tag from line 3 to line 10